packages/build/src/plugins/spawn.ts

```diff
@@ -50,6 +50,8 @@ const startPlugin = async function ({ pluginDir, nodePath, buildDir, childEnv, s
     preferLocal: true,
     localDir: pluginDir,
     nodePath,
+    // make sure we don't pass build's node cli properties for now (e.g. --import)
+    nodeOptions: [],
```

Comment:
While testing out locally I've spotted that  `execaNode` inherits the nodeOptions we pass to `build`. For now we want to make sure this is disabled as to not cause any issues with plugin execution.